streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -600,24 +562,7 @@ public void shouldNotSetInternalAutoDowngradeTxnCommitToTrueInProducerForEosDisa
         assertThat(producerConfigs.get("internal.auto.downgrade.txn.commit"), is(nullValue()));
     }
 
-    @SuppressWarnings("deprecation")
-    @Test
-    public void shouldSetInternalAutoDowngradeTxnCommitToTrueInProducerForEosAlpha() {
-        props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE);
-        final StreamsConfig streamsConfig = new StreamsConfig(props);
-        final Map<String, Object> producerConfigs = streamsConfig.getProducerConfigs(clientId);
-        assertThat(producerConfigs.get("internal.auto.downgrade.txn.commit"), is(true));
-    }
-
-    @SuppressWarnings("deprecation")
-    @Test
-    public void shouldNotSetInternalAutoDowngradeTxnCommitToTrueInProducerForEosBeta() {
-        props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE_BETA);
-        final StreamsConfig streamsConfig = new StreamsConfig(props);
-        final Map<String, Object> producerConfigs = streamsConfig.getProducerConfigs(clientId);
-        assertThat(producerConfigs.get("internal.auto.downgrade.txn.commit"), is(nullValue()));
-    }
-
+    // TODO: should we keep this test? (cf other TODO)
```

Comment:
Relates to the question above, about the remove code which sets the internal config.